worker/src/logger.ts

```diff
@@ -1,33 +0,0 @@
-import pino from "pino";
-import { env } from "./env";
-
-const getBetterstackLogginTransport = (minLevel: string) => {
```

Comment:
When deploying this, we should remove the LANGFUSE_WORKER_BETTERSTACK_TOKEN from our environment. Can you also remove it from everywhere else in the repo?

Reply:
Addressed in https://github.com/langfuse/langfuse/pull/3221/commits/1b2409018704dbc34d10aac1b67db2ede6b5ed77.